test/solc/CommandLineParser.cpp

```diff
@@ -183,11 +185,12 @@ BOOST_AUTO_TEST_CASE(cli_mode_options)
 		expectedOptions.output.evmVersion = EVMVersion::spuriousDragon();
 		expectedOptions.output.experimentalViaIR = true;
 		expectedOptions.output.revertStrings = RevertStrings::Strip;
+		using util::JsonFormat;
+		expectedOptions.formatting.json = JsonFormat{JsonFormat::Pretty, 7};
```

Comment:
There's `using namespace solidity::util` at the top of the file so you can access `JsonFormat` without the extra `using` here.
```suggestion
		expectedOptions.formatting.json = JsonFormat{JsonFormat::Pretty, 7};
```

Same in two places below.